CodeStarterCamp_Week4/Activity.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Step1.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by Whales on 2023/03/17.
+//
+
+import Swift
+
+struct Activity {
+    let name: String
+    let action: (BodyCondition) -> Void
+}
+
+let bodyCondition: BodyCondition = BodyCondition(upper: 10, lower: 15, muscular: 20, fatigue: 30)
+let sitUp: Activity = Activity(name: "윗몸일으키기", action: { bodyCondition in
+    let effect1 = Int.random(in: 10...20)
+    let effect2 = Int.random(in: 10...20)
```

Comment:
bodyCondition의 근력을 바로 올려줄 수 있지 않을까요? 
```suggestion
    bodyCondition.upperBodyMuscleStrength += Int.random(in: 10...20)
    bodyCondition.degreeOfFatigue += Int.random(in: 10...20)
```

Reply:
처음에는 바로 올렸다가 프린트 할 때 얼만큼 올랐는지 차이를 보여줘야 하더라고요
그래서 분리시켰어요!!
그러면 바로 올리고 프린트를 할때 차이를 보여주는 걸로 수정할까요?
아 근데 그렇게 되면 `didSet`이랑 `print문`의 순서도 바뀔거라 그 부분 수정도 고려해봐야 할 것 같아요!!